packages/astro/src/vite-plugin-integrations-container/index.ts

```diff
@@ -11,10 +11,28 @@ export default function astroIntegrationsContainerPlugin({
 	settings: AstroSettings;
 	logging: LogOptions;
 }): VitePlugin {
+	let hasResolvedInjectedRoutes = false;
 	return {
 		name: 'astro:integration-container',
 		configureServer(server) {
 			runHookServerSetup({ config: settings.config, server, logging });
 		},
+		// Run `load` hook once just to get access to `PluginContext.resolve`
+		// which allows us to resolve each injectedRoute entryPoint to a real file URL on disk
+		async load() {
```

Comment:
I think we can also call this in `buildStart()`?

Reply:
Great call, thanks!